deps/rabbitmq_prometheus/src/collectors/prometheus_rabbitmq_core_metrics_collector.erl

```diff
@@ -240,11 +240,11 @@ collect(PerObjectMetrics, Callback) ->
     add_metric_family(identity_info(), Callback),
     ok.
 
-needs_processing(false, auth_attempt_detailed_metrics) ->
-    %% When per object metrics are disabled the detailed authentication attempt metrics
-    %% create duplicates. Totals are carried on `auth_attempt_metrics`
+include_when_per_object_metrics(true, auth_attempt_metrics) ->
```

Comment:
If we exclude the non-detailed totals and the user has not set `track_auth_attempt_source` to `true` it will end up without metrics. This is a misconfiguration issue, but it makes me wonder if we should always report the globals anyway. We only need to use different keys for the values (such as `auth_attempts_detailed_total`), and hide them when not required. @michaelklishin @gerhard thoughts?

Reply:
I like that idea very much! Having a go at it now.